packages/amplication-client/src/Workspaces/hooks/useResources.ts

```diff
@@ -47,6 +48,21 @@ const useResources = (
     skip: !currentProject,
   });
 
+  const resourceRedirect = useCallback(
+    (resourceId: string) =>
+      history.push({
+        pathname: `/${currentWorkspace?.id}/${currentProject?.id}/${resourceId}`,
+      }),
+    [currentWorkspace, history, currentProject]
+  );
+
+  const onNewResourceCompleted = useCallback(
```

Comment:
I don't understand why you need to expose this outside of the hook. It's a straight forward event that happen after creating new resource

Reply:
Done